Sources/Redis/RedisConfiguration.swift

```diff
@@ -49,7 +49,7 @@ public struct RedisConfiguration {
             hostname: url.host ?? "localhost",
             port: url.port ?? RedisConnection.defaultPort,
             password: url.password,
-            database: Int(url.path),
+            database: Int(url.lastPathComponent),
```

Comment:
@gustavoggs Could you write a unit test for this?

Reply:
@Mordil sure, I will try to update this later today